### PR TITLE
Lint fix

### PR DESCRIPTION
**Pull request by dansweet1**
Lint error free

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,24 +4,35 @@
   "description": "",
   "main": "lib/index.js",
   "scripts": {
-    "test": "node -r babel-register -r babel-polyfill -r dotenv/config test/test.js | faucet",
-    "tail": "tail -f app.log | bunyan -o short"
+      "test": "mocha --require babel-polyfill --compilers js:babel-register",
```
Comment (perryao) [inline package.json]: indentation looks slightly off here.

File: .eslintrc [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-	"indent": [1, 4],
-    },
     "plugins": ["babel"],
-    "parser": "babel-eslint"
-}
\ No newline at end of file
+    "parser": "babel-eslint",
+    "rules": {
+      "indent": ["error", 4],
+      "no-unused-vars": 0,
```
Comment (perryao) [inline .eslintrc]: this is one we shouldn't disable. No point in declaring a var if it isn't used.

File: lib/index.js [hunk 3]
```diff
@@ -49,9 +45,9 @@ export default class GithubSource extends Activity {
         return assignees;
     }
 
-    async listIssueComments({ user, repo }, gh) {
+    async listIssueComments({ user, repo, number }, gh) {
```
Comment (perryao) [inline lib/index.js]: @dansweet1 isn't this change from your number param branch? Ideally, this pr would only have the changes from the lint fixes. We can close the numberParam branch and pr though if those changes are in here now.

**Comment by perryao** [comment]
@dansweet1 looks like the tests are still written with blue-tape, which is the only reason the `npm test` or `npm coverage` commands are passing right now. Since you removed the static props, will also need to get rid of the test for the props. And since you changed the test command to use mocha instead of blue-tape, you'll also need to rewrite the other existing test in mocha format and remove the blue-tape test dependencies.

**Comment by dansweet1** [comment]
Ben told me that Julian is already re-writing all tests using Mocha which
is why I did not refactor them. Should I still proceed since they are going
to be re-written?

**Comment by perryao** [comment]
Nope but be sure to loop @jlaneve in so he knows this is getting merged first.

**Comment by dansweet1** [comment]
Will do!

**Comment by perryao** [comment]
alright looks good to me. Last thing to think about is whether you removing static props is going to conflict with anything @jlaneve  is doing. We don't need those anymore, so they should be removed. But he may already have done that since he's working on mocha testing.

**Pull request merged**